yt/yt/client/table_client/wire_protocol.h

```diff
@@ -280,13 +280,24 @@ struct IWireProtocolReader
 
 ////////////////////////////////////////////////////////////////////////////////
 
+struct TWireProtocolReaderLimits {
+    ui32 MaxStringValueLength = NYT::NTableClient::MaxStringValueLength;
```

Comment:
Also, maybe we want to make all of these values signed? We actually discourage using unsigned types for non-negative types (similar to Google C++ code style), and the types of these constants are just legacy. Unsigned types should be used for bitwise-arithmetic-related things.

Reply:
These values are compared to other values in the code. I keep types the same

`MaxStringValueLength` is `ui32` because variable we compare to is `ui32`
https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/client/table_client/wire_protocol.cpp#L860

Same for others

Reply:
Meh, ok, fine.